mosaic2/grid2.F90

```diff
@@ -600,7 +600,7 @@ subroutine get_grid_comp_area_SG(component,tile,area,domain)
         deallocate(nest_tile_name)
      end select ! version
      ! convert area to m2
-     area = area*4.*PI*radius**2
+     area = real(area*4.*PI*radius**2, r4_kind)
```

Comment:
``` fortran
cellarea = real(cellarea*4.*PI*radius**2., r4_kind)

```

Reply:
The 2 is an exponent, so making it a real could change answers due to f-p representations for "2" vs "2."